packages/xstate-wallet/src/index.ts

```diff
@@ -5,14 +5,16 @@ import {MessagingService} from './messaging';
 import {ChainWatcher} from './chain';
 import {IndexedDBBackend} from './store/indexedDB-backend';
 import {MemoryBackend} from './store/memory-backend';
-import {TestStore} from './workflows/tests/store';
+import {XstateStore} from './store';
 
 (async function() {
   const {privateKey} = ethers.Wallet.createRandom();
   const chain = new ChainWatcher();
+
   const backend = process.env.USE_INDEXED_DB ? new IndexedDBBackend() : new MemoryBackend();
-  const store = new TestStore(chain, backend);
-  await store.initialize([privateKey]);
+  const store = new XstateStore(chain, backend);
+  const cleanSlate = !!process.env.CLEAR_STORAGE_ON_START || false;
```

Comment:
In recent discussions, we have concluded that it might be a good pattern to never access `process.env` keys outside of `constants.ts`. So in this case, `cleanState` would be defined in and exported by `constants.ts`. 

It also might be good to use [`getEnvBool`](https://github.com/statechannels/monorepo/blob/master/packages/devtools/src/config/env.ts#L4) for boolean environment variables.